core/docker/default/apt/sources.list.d/mirrors-amd64.sources

```diff
@@ -0,0 +1,6 @@
+Enabled: yes
+Types: deb
+URIs: https://mirrors.ocf.berkeley.edu/ubuntu/ https://mirror.kumi.systems/ubuntu/
+Suites: jammy jammy-updates jammy-backports jammy-security
```

Comment:
Will it always be `jammy`. Aren't they bumping debian version when there is new one in `eclipse-temurin:17-jdk`?

Reply:
I think once `eclipse-temurin:17-jdk` updates their base image we should update the mirror descriptors manually.
Other than pinning ourselves to a specific hash of `eclipse-temurin` we have no control over the distro version they use.
They might even switch from Ubuntu to something which doesn't use APT.

Reply:
The point is we will not notice that moment right?
Or will we get some error when they update?

Reply:
The error might not directly mention a mismatch of Ubuntu versions, but I expect it might look like a huge update with APT trying to pull every installed package twice.

Reply:
I also wonder if using a tag instead of a hash for the base image is a good idea. I think it's better to use immutable hashes for build repeatability.

Reply:
We can probably build these mirrorlists dynamically during image build by something like:

```bash
source /etc/os-release
# the above exports a UBUNTU_CODENAME variable
sed -i 's/UBUNTU_CODENAME/${UBUNTU_CODENAME}/g' /etc/apt/sources.list.d/*
```

Reply:
>  I also wonder if using a tag instead of a hash for the base image is a good idea

I think the point of not using specific version it to always get the most recent security patches in place

> We can probably build these mirrorlists dynamically during image build by something like

This looks nice

Reply:
> > I also wonder if using a tag instead of a hash for the base image is a good idea
> 
> I think the point of not using specific version it to always get the most recent security patches in place

I understand that was the intention, but it seems risky because a build of the same trino git hash might work today and fail tomorrow just because the tag was updated in between. I think a more repeatable approach would be to pin ourselves to a specific docker image hash and use `dependabot` to keep it up to date with the tag. In this case every docker image change would correspond with a commit in trino that we can roll back if it breaks something.